frontend/static/js/select-2.js

```diff
@@ -0,0 +1,55 @@
+Vue.component('select-2', {
+  template: '<select v-bind:name="name" class="form-control" v-bind:multiple="multiple"></select>',
+  props: {
+    name: '',
+    options: {
+      Object
+    },
+    value: null,
+    multiple: {
+      Boolean,
+      default: false
+
+    }
+  },
+  data() {
+    return {
+      select2data: [],
```

Comment:
can we rename the fields to more meaningful names that tell us what they contain rather than what control they're bound to?

Reply:
rename it to `optionsData` because `options` already used as props.